MapboxSearch/gradle/versions.gradle

```diff
@@ -45,7 +45,7 @@ ext {
     common_sdk_version = '24.0.0'
     mapbox_base_version = '0.8.0'
 
-    search_native_version = '2.0.0-alpha.4'
+    search_native_version = '2.0.0-alpha.11'
```

Comment:
I believe @ahmedaly16 mentioned that we need to get to alpha.13

Reply:
Done